docs/gatsby-config.js

```diff
@@ -64,8 +64,9 @@ module.exports = {
             'advanced/no-runtime',
           ],
           Testing: [
-            'advanced/ui-tests',
-            'advanced/test-builders',
+            'testing/ui-tests',
```

Comment:
Not strictly necessary but I thought it would make sense to move these pages to their own folder (to be consistent with the structure of the doc site). Added redirections in `static/_redirects`  (hoping it will work!)

Reply:
Agree this is cleaner 👍 

If we're rehauling this, I'd suggest changing the names a little bit and adding an `overview` page like for normalized cache:

* Overview
* Mock HTTP requests
* Mock GraphQL requests
* Test builders
* Espresso IdlingResource

The overview page would explain that it's possible to test end-to-end with a mock HTTP server. In that case, the same ApolloClient can be used for tests and production. Or for lighter tests that don't need to create a server, it's possible to use a MockNetworkTransport.

* For GraphQL tests, the Test Builders can produce an `Operation.Data`.
* For HTTP tests, the Test Builders can produce an Json String (this part still needs to be done but it shouldn't be too hard).

Reply:
The redirects look like they'll look great! 😄

Reply:
Agreed! I'll do these changes now.